src/gepetto_viewer_rerun/client.py

```diff
@@ -74,27 +74,93 @@ def createScene(self, sceneName: str):
         )
         logger.info(msg)
 
-    def _getSceneIndex(self, sceneName: str):
-        for index, scene in enumerate(self.sceneList):
+    def getWindowList(self):
+        return [scene.name for scene in self.windowList]
```

Comment:
this is returning the scene name list, not a window list.

Reply:
Despite the bad variable name (scene instead of window) it still iterate over `self.windowList`